_data/chains/eip155-65.json

```diff
@@ -3,12 +3,12 @@
   "chain": "okexchain",
   "network": "testnet",
   "rpc": [
-    "https://www.okex.com/okexchain-rpc-testnet"
+    "https://exchaintest.okexcn.com"
   ],
   "faucets": ["https://www.okex.com/drawdex"],
   "nativeCurrency": {
     "name": "OKExChain Global Utility Token in testnet",
-    "symbol": "TOKT",
+    "symbol": "OKT",
```

Comment:
I think it is good that the testnet has a different symbol than the mainnet. Don't you think?
Rest of the PR looks good

Reply:
> I think it is good that the testnet has a different symbol than the mainnet. Don't you think?
> Rest of the PR looks good

Yes, I agree with you, but we have lanuched testnet with OKT symbol.
Thank you for your kind help.